src/fstar/FStar.Main.fst

```diff
@@ -133,6 +133,8 @@ let go _ =
     | Success ->
         fstar_files := Some filenames;
 
+        load_native_tactics ();
```

Comment:
load_native_tactics early, since now even the early phases of the parser, dependence scanning etc. can depend on a native plugin